test/unit/state/siteUtilTest.js

```diff
@@ -414,6 +414,50 @@ describe('siteUtil', function () {
     })
   })
 
+  describe('isHistoryEntry', function () {
+    it('returns true for a typical history entry', function () {
```

Comment:
not sure if someone else reading this will understand what makes it "typical". Maybe "returns true for site entries with a lastAccessedTime"?